src/Data/Languages.hs

```diff
@@ -27,7 +32,7 @@ languagesForPath path = languageForFileName <|> languageForExtension
     languageForFileName = languageFor (takeFileName path) languagesByFileName
     languageForExtension = languageFor (takeExtension path) languagesByExtension
     languageFor :: String -> Map.Map Text [LanguageKey] -> [Language]
-    languageFor k
-      = foldMap (maybeToList . flip Map.lookup languages)
-      . fromMaybe []
-      . Map.lookup (Text.pack k)
+    languageFor k =
+      foldMap (maybeToList . flip Map.lookup languages)
+        . fromMaybe []
+        . Map.lookup (Text.pack k)
```

Comment:
There’s some weird reformatting that I’m not sure was intentional?

Reply:
Intentional. I hit this with `ormolu`, since it (once applied) tends to make for significantly better diffs; both @tclem and I are fans.